certbot/cli.py

```diff
@@ -898,12 +913,21 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
 def _create_subparsers(helpful):
     helpful.add("config_changes", "--num", type=int,
                 help="How many past revisions you want to be displayed")
+
+    class DummyConfig(object):
```

Comment:
This takes advantage of `determine_user_agent`'s implementation details. If we wanted to change those, we have to change this here, which is weird. Could this be a new method in `client.py` called `demo_user_agent`, or something to that effect that keeps the details of what goes into a user agent in `client.py`?